apps/ciphers.c

```diff
@@ -34,7 +34,7 @@ const OPTIONS ciphers_options[] = {
     OPT_SECTION("General"),
     {"help", OPT_HELP, '-', "Display this summary"},
 
-    OPT_SECTION("Output"),
+    OPT_SECTION("Cipher specification"),
```

Comment:
Hmm, should there be two separate groups? Output (-v, -V, -stdname, -convert) and Cipher selection (for the rest)?

Reply:
yes, there should :)  additional commit pushed.  thanks.